spec/integration/operations_spec.rb

```diff
@@ -134,9 +134,20 @@ module Hawkular::Operations::RSpec
       end
 
       describe 'Operation/Operation' do
-        before(:all) do
-          @random_uuid = SecureRandom.uuid
-          @not_so_random_uuid = 'not_so_random_uuid'
+        let(:random_uuid) do
+          SecureRandom.uuid
+        end
+
+        let (:not_so_random_uuid) do
```

Comment:
I think @cfcosta  is saying that you can remove `not_so_random_uuid`. Instead, use `random_uuid` everywhere and the tests will be fine.